backend/src/services/authService.js

```
@@ -1,6 +1,7 @@
 const { AppDataSource } = require('../config/database');
 const User = require('../models/User');
 const bcrypt = require('bcrypt');
+const crypto = require('crypto');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)